frontend/src/__tests__/cypress/cypress/e2e/projects/ProjectList.cy.ts

```diff
@@ -108,6 +132,113 @@ describe('Data science projects details', () => {
     projectListPage.findProjectLink('DS Project 2').should('not.exist');
     projectListPage.findProjectLink('renamed').should('not.exist');
   });
+
+  describe('Table filter', () => {
+    it('filter by name', () => {
+      initIntercepts();
+      projectListPage.visit();
+
+      // Verify initial run rows exist
+      projectListPage.findProjectListRow('Test Project').should('have.length', 1);
+
+      // Select the "Name" filter
+      const projectListToolbar = projectListPage.getTableToolbar();
+      projectListToolbar.findFilterMenuOption('filter-dropdown-select', 'Name').click();
+      projectListToolbar.findSearchInput().type('Test Project');
+      // Verify only rows with the typed run name exist
+      projectListPage.findProjectListRow('Test Project').should('exist');
+    });
+
+    it('filter by user', () => {
+      initIntercepts();
+      projectListPage.visit();
+
+      // Verify initial run rows exist
+      projectListPage.findProjectListRow('Test Project').should('have.length', 1);
+
+      // Select the "User" filter
+      const projectListToolbar = projectListPage.getTableToolbar();
+      projectListToolbar.findFilterMenuOption('filter-dropdown-select', 'User').click();
+      projectListToolbar.findSearchInput().type('test-user');
+      // Verify only rows with the typed run user exist
+      projectListPage.findProjectListRow('Test Project').should('exist');
+    });
+  });
+
+  it('Validate that clicking on switch toggle will open modal to stop workbench', () => {
+    cy.interceptK8sList(ProjectModel, mockK8sResourceList([mockProjectK8sResource({})]));
+    cy.interceptK8s('PATCH', NotebookModel, mockNotebookK8sResource({})).as('stopWorkbench');
+    cy.interceptK8sList(PodModel, mockK8sResourceList([mockPodK8sResource({})]));
+    cy.interceptK8s(RouteModel, mockRouteK8sResource({ notebookName: 'test-notebook' })).as(
+      'getWorkbench',
+    );
+    cy.interceptK8sList(
+      { model: NotebookModel, ns: 'test-project' },
+      mockK8sResourceList([
+        mockNotebookK8sResource({
+          opts: {
+            spec: {
+              template: {
+                spec: {
+                  containers: [
+                    {
+                      name: 'test-notebook',
+                      image: 'test-image:latest',
+                    },
+                  ],
+                },
+              },
+            },
+            metadata: {
+              name: 'test-notebook',
+              labels: {
+                'opendatahub.io/notebook-image': 'true',
+              },
+              annotations: {
+                'opendatahub.io/image-display-name': 'Test image',
+              },
+            },
+          },
+        }),
+      ]),
+    );
+    projectListPage.visit();
+    cy.wait('@getWorkbench');
+    const projectTableRow = projectDetails.getProjectTableRow('Test Project');
```

Comment:
can we just use `projectListPage.getProjectRow('Test Project')` instead of `projectDetails.getProjectTableRow()`
```suggestion
    const projectTableRow = projectListPage.getProjectRow('Test Project');
```